apps/web/app/(app)/(survey-editor)/environments/[environmentId]/surveys/[surveyId]/edit/components/HtmlHeader.tsx

```
@@ -0,0 +1,40 @@
+
+  if (!htmlString) return null;
+  if (safeHtml === `<p class="fb-editor-paragraph"><br></p>`) return null;
+
+  return (
+    <label
+      htmlFor={questionId}
+      className={cn(
+        "fb-break-words flex text-sm font-semibold",
+        "[&_span]:inline-block [&_span]:align-baseline" // Add specific styling for spans
+      )}
+      dangerouslySetInnerHTML={{ __html: safeHtml }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)